.gitignore

```diff
@@ -38,6 +38,9 @@ target
 # .vscode
 *.vscode
 
+# .idea
+*.idea
+
```

Comment:
there's already an entry for this
```suggestion
```

Reply:
fixed in https://github.com/wso2/apk/pull/151/commits/4f4c37b97abe8646e783d357a3998601be6de56c